svelte-compiler.js

```diff
@@ -2,11 +2,7 @@ const htmlparser = Npm.require('htmlparser2');
 const sourcemap = Npm.require('source-map');
 const svelte = Npm.require('svelte-es5-meteor');
 
-Plugin.registerCompiler({
-  extensions: ['html'],
-}, () => new SvelteCompiler);
-
-class SvelteCompiler extends CachingCompiler {
+SvelteCompiler = class extends CachingCompiler {
```

Comment:
Exported variables (see *package.js*) have to be global. `class SvelteCompiler …` would be transpiled to a local variable: `var SvelteCompiler = function (_CachingCompiler) {`.